tests/rip7560/conftest.py

```diff
@@ -58,10 +59,27 @@ def wallet_contract_rules(w3):
 def tx_7560(wallet_contract):
     return TransactionRIP7560(
         sender=wallet_contract.address,
+        nonceKey=hex(0),
         nonce=hex(1),
         maxFeePerGas=hex(100000000000),
         maxPriorityFeePerGas=hex(100000000000),
         verificationGasLimit=hex(2000000),
         callData=wallet_contract.encodeABI(fn_name="anyExecutionFunction"),
         signature="0xface",
     )
+
+
+@pytest.fixture(scope="session")
+def nonce_manager(w3):
+    current_dirname = os.path.dirname(__file__)
+    nonce_manager_artifact_path = os.path.realpath(
```

Comment:
our test code uses compile_contract, instead of artifact. I think this line is the reason you need to compile as a update_dep dependency.